web/cypress/e2e/auto-tax-filings.spec.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  clickModalSaveButton,
+  completeBusinessStructureTask,
+  openFormationDateModal,
+  selectDate,
+  selectLocation,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { completeNewBusinessOnboarding } from "@businessnjgovnavigator/cypress/support/helpers/helpers-onboarding";
+import { LookupIndustryById } from "@businessnjgovnavigator/shared/";
+import { onDashboardPage } from "cypress/support/page_objects/dashboardPage";
+
+describe("auto tax filing [feature] [all] [group4]", () => {
```

Comment:
dust - cool that you added a feature test, I would definitely ping marques if you haven't for his awareness as he wanted to do something of an overhaul, so it's just good if he knows when things have appeared.

Reply:
@mjackson21 @FarazA22